internal/host/plugin/ids.go

```diff
@@ -19,24 +18,22 @@ func init() {
 // PublicId prefixes for the resources in the plugin package.
 const (
 	// TODO: Pull these out of being constants and have them derivable at run time.
-	HostCatalogPrefix = "hcplg"
-	HostSetPrefix     = "hsplg"
+	HostCatalogPrefix = "hc"
```

Comment:
I think rather than have this here we should just move it to `globals` somewhere (maybe a new file) so that as other resources are modified they can all reference the same spot.

Reply:
I started down that path but decided to wait until we did this more broadly.  I'm hoping to avoid discussions around what structure we want to change to since this is already a decently sized PR.

Reply:
OK